mobile/src/components/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { FC } from 'react';
+import { Image } from 'react-native';
```

Comment:
Re-export it
```suggestion
import { Image } from 'components/common/common';
```

Reply:
Do we have 'Image' in components/common? Or I should create it?

Reply:
Yes, re-export it from there

Reply:
If it is not exists, create

Reply:
![Screenshot_2](https://user-images.githubusercontent.com/89215185/183893584-6651cb07-bea6-45dc-83aa-1aa4224573c6.png)

you can hit me, but I don't see it there))

Reply:
If you cannot see it, just create

Reply:
or i should add Import of Image from react-native to common file? and than import it from there?